core/commonMain/src/kotlinx/io/bytes/BytesOperations.kt

```diff
@@ -0,0 +1,25 @@
+package kotlinx.io.bytes
+
+import kotlinx.io.*
+import kotlinx.io.buffer.*
+import kotlin.contracts.*
+
+/**
+ * Create [BytesInput] with content from [block] using specified [bufferSize].
+ */
+fun buildInput(bufferSize: Int = DEFAULT_BUFFER_SIZE, block: BytesOutput.() -> Unit): BytesInput {
+    contract {
+        callsInPlace(block, InvocationKind.EXACTLY_ONCE)
+    }
+
+    return BytesOutput(bufferSize).apply {
+        block()
+    }.createInput()
+}
+
+/**
+ * Read [BytesInput] of fixed [size].
+ */
+public fun Input.readBytesInput(size: Int): BytesInput = buildInput {
```

Comment:
Such family of operators is typically named with prefix `to` in Kotlin.
Please stick with an official naming scheme